<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgGlobalTransform</class>
 <widget class="QDialog" name="DlgGlobalTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Translate X</string>
     </property>
     <property name="buddy">
      <cstring>m_sbTranslateX</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QScienceSpinBox" name="m_sbTranslateX">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-99999999999999991433150857216.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000000000019884624838656.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Translate Y</string>
     </property>
     <property name="buddy">
      <cstring>m_sbTranslateX</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QScienceSpinBox" name="m_sbTranslateY">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-99999999999999991433150857216.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000000000019884624838656.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Translate Z</string>
     </property>
     <property name="buddy">
      <cstring>m_sbTranslateX</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QScienceSpinBox" name="m_sbTranslateZ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-99999999999999991433150857216.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000000000019884624838656.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QScienceSpinBox" name="m_sbScale">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-99999999999999991433150857216.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000000000019884624838656.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pbApply">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbClose">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbApply</sender>
   <signal>clicked()</signal>
   <receiver>DlgGlobalTransform</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DlgGlobalTransform</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
